nuget/lib/dependabot/nuget/nuget_config_credential_helpers.rb

```diff
@@ -55,17 +55,16 @@ def self.restore_user_nuget_config
         File.rename(temporary_nuget_config_path, user_nuget_config_path)
       end
 
-      # rubocop:disable Lint/SuppressedException
       def self.patch_nuget_config_for_action(credentials, &_block)
         add_credentials_to_nuget_config(credentials)
         begin
           yield
-        rescue StandardError
+        rescue StandardError => e
+          puts "block causes an exception #{e}: #{e.message}"
```

Comment:
Should this use `Dependabot.logger` instead of `puts`?

Reply:
Logger, in general, sounds good. I just did not see it used anywhere (I am monkey writing Ruby I do not know the language or tools) hence I did not use it :)